include/gatekeeper_net.h

```diff
@@ -212,6 +214,14 @@ struct gatekeeper_if {
 	 */
 	acl_cb_func        acl_funcs[GATEKEEPER_IPV6_ACL_MAX];
 
+	/*
+	 * Callback functions for each ACL rule type with
+	 * IPv6 extension headers.
+	 *
+	 * On error, these functions should return a negative value.
```

Comment:
I think we should mention here that returning 0 means a match and returning a negative value means an error or that there was no match.